wiki/Article_Styling_Criteria/en.md

```diff
@@ -48,39 +48,43 @@ Listed below are the properly-supported locales for the osu! wiki. The file name
 | `zh.md` | Chinese (Simplified) | 简体中文 |
 | `zh-tw.md` | Traditional Chinese (Taiwan) | 繁體中文（台灣） |
 
-*Notice: It is intended that the osu! wiki will try to give readers their selected language's version of an article (and fallback on using the English version if it is not available).*
+*Note: The osu! wiki will give readers their selected language's version of an article, but will fallback on the English version if it is not available.*
 
-If your language is not listed above, you can either append `?locale={langcode}` to the URL (where `{langcode}` is your language's two letter code in lowercase letters, e.g. `https://osu.ppy.sh/help/wiki/Welcome?locale=zh` will give you the Chinese version of the Welcome article regardless of the selected language) or help translate the [osu-web via Crowdin](https://crowdin.com/project/osu-web). If you need help translating the website or want to discuss translating decisions, you can talk about it in the [osu!dev Discord](https://discord.gg/ppy) (`#osu-web` channel).
+If your language is not listed above, you can either append `?locale={langcode}` to the URL (where `{langcode}` is your language's two letter code in lowercase letters, e.g. `https://osu.ppy.sh/help/wiki/Welcome?locale=zh` will give you the Chinese version of the Welcome article regardless of the selected language) or help translate the [osu-web via Crowdin](https://crowdin.com/project/osu-web). If you need help translating the website or want to discuss translating decisions, you can discuss about it in the [osu!dev Discord](https://discord.gg/ppy) (`#osu-web` channel).
 
 ### Metadata
 
-*Notice: Metadata must be placed at the very top of the file; otherwise, it will be parsed as plain text.*
+Metadata must be placed at the very top of the file. An empty line must follow it.
 
 #### Outdated articles
 
-*Note to translators: After updating an article, remove the `outdated` mark.*
+*Note to translators: The `outdated` mark must be removed after updating the translation of an article.*
 
-Use the `outdated` mark on translated articles when the English variant is updated, which looks like this:
+Translated articles that are outdated must use the `outdated` mark when the English variant is updated. The markup is as follows:
 
 ```md
 ---
 outdated: true
 ---
+
+# Beatmaps
 ```
 
 #### Tagging articles
 
-*Note: You should only need to include words that are not found in the article.*
-
-Tagging articles will help the website's search engine query articles better. It is unlikely that you will need to do this, but the markup is as follows:
+Tagging articles will help the website's search engine query articles better. Tags should use lowercase letters. It may be unlikely that this is needed for most articles; however, the markup is as follows:
 
 ```md
 ---
 tags:
```

Comment:
Nit: I think since this is a front matter with a YAML syntax, we should have it as `yml` instead.

Reply:
Down below on line 87, I included a title to show that a space is required. Changing it to `yaml` will make that look like a comment instead. (keeping it as-is)